src/Workspace/Wes_Timing_System_4/TimingScript/TimingCommandsService.lua

```diff
@@ -0,0 +1,44 @@
+local Workspace = game:GetService("Workspace")
+local _Config = require(Workspace.Wes_Timing_System_4._Config)
+local TimingDataService = require(script.Parent.TimingDataService)
+
+local service = {}
+
+function service.AddCommandsToChat(plr: Player)
+    for _,v in pairs(_Config.AdminUsers) do
+        if plr.Name == v then
+            plr.Chatted:Connect(function(msg)
+                local loweredString = string.lower(msg)
+                local args = string.split(loweredString," ")
+
+                if args[1] == ":setlaps" then
+                    print("Setting Laps!")
+                    for _,player in pairs(game:GetService("Players"):GetPlayers()) do
+                        if (string.sub(string.lower(player.Name), 1, string.len(args[2])) == string.lower(args[2])) 
```

Comment:
I don't know how other admin scripts do it, but I would prioritize DisplayName over Name because the former is what most people will probably try to type first.

Also I would create a separate function for matching the input to a player instead of having duplicate code:
```lua
// Returns null if no player is found, else return Player instance
function SearchPlayer(input: string): Player | null
   // Implementation
end
```